### ISSUE TITLE
Add TalkBack actions menu support for compose field

### ISSUE THREAD

**Issue by alexstine** [signalapp/Signal-Android#13267]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description

If you compose a message in Signal, there is no way with TalkBack via TalkBack actions to select all and delete. Developer docs can be found below.

https://developer.android.com/guide/topics/ui/accessibility/principles#accessibility-actions

### Steps to reproduce
- Enable TalkBack: https://support.google.com/accessibility/android/answer/6007100?hl=en.
- Open Signal.
- Select an existing chat.
- Compose a message.
- Explore with touch until you find the compose box with your composed message.
- Swipe left or right with three fingers to cycle through available TalkBack actions. Notice how words, characters, lines, headings, and spell check are available, but not edit. Your actions may differ based on what you have selected in TalkBack settings, but edit should not be missing. Edit allows you to move your cursor, select all, etc.
- Alternatively, swipe up then right to open the TalkBack menu. See that the first item, Actions, is missing.

**Actual result:** Describe here what happens after you run the steps above (i.e. the buggy behaviour)
The edit action is not available.

**Expected result:** Describe here what should happen after you run the steps above (i.e. what would be the correct behaviour)
The edit action should be available.

### Device info

**Device:** Google Pixel 7
**Android version:** 14
**Signal version:** 6.40.5

**Comment by alexstine** [comment]
Related: https://github.com/signalapp/Signal-Android/issues/7934#issuecomment-493728706.

**Comment by alexstine** [comment]
Still an issue.

**Comment by jonathans859** [comment]
Are there any updates on this? Just realised this too and it's bad.
